x-pack/plugins/maps/public/ems_autosuggest/ems_autosuggest.ts

```diff
@@ -0,0 +1,204 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { FileLayer } from '@elastic/ems-client';
+import { getEmsFileLayers, fetchGeoJson } from '../util';
+import { FORMAT_TYPE } from '../../common';
+
+export interface ISampleValuesConfig {
```

Comment:
Why do these interface names have `I` prefix? We have been using `I` prefix to identify class interface that gets implementations. Since these are just a collection of properties, how about removing the `I` prefix?